### PR TITLE
Fix spelling for the ExceptionCatcherProcessor.

### PR DESCRIPTION
**Pull request by yoyosan**

### CODE REVIEW

File: src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php [hunk 1]
```diff
@@ -6,6 +6,11 @@
 use Swarrot\Processor\ProcessorInterface;
 use Psr\Log\LoggerInterface;
 
+/**
+ * Class ExceptionCatcherProcessor
```
Comment (stof) [inline src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php]: this provides exactly no value

File: src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php [hunk 2]
```diff
@@ -6,6 +6,11 @@
...
+ *
+ * @package Swarrot\Processor\ExceptionCatcher
```
Comment (stof) [inline src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php]: ``@package`` is not used in Swarrot

File: src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php (hunk 1 above)
Comment (yoyosan) [inline src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php]: I'm trying to follow the http://symfony.com/doc/current/contributing/code/standards.html#documentation

File: src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php (hunk 2 above)
Comment (yoyosan) [inline src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php]: Is there a coding guideline somewhere because I didn't find any?

Comment (odolbeau) [inline src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php]: There is definitively a lack of documentation regarding swarrot. I need to take time to write some & add contributing information. :/
As said by @stof, we don't use the `@package` annotation in swarrot and the doc block you add on this class doesn't not provide any value, can you remove it?
Thanks for your contribution. :)

Comment (yoyosan) [inline src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php]: Done @odolbeau. Do you think I should have a look into all READMEs and messages?

Comment (odolbeau) [inline src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php]: Feel free to do so if you want. :)

**Comment by behinddesign** [comment]
This improves the readability of the exception message.

**Comment by odolbeau** [comment]
Thanks! Sorry for the delay. :/

**Pull request merged**